src/Confluent.Kafka/AdminClient.cs

```diff
@@ -981,6 +1009,23 @@ private Task StartPollTask(CancellationToken ct)
                                         
                                         break;
                                     }
+                                    case Librdkafka.EventType.ListOffsets_Result:
+                                    {
+                                        if (errorCode != ErrorCode.NoError)
+                                        {
+                                            Task.Run(() =>
+                                                    ((TaskCompletionSource<Null>)adminClientResult).TrySetException(
+                                                        new KafkaException(kafkaHandle.CreatePossiblyFatalError(errorCode, errorStr))));
+                                                break;
+                                        }
+                                        var results = extractListOffsetsResults(eventPtr);
+                                        Task.Run(() =>
+                                                ((TaskCompletionSource<ListOffsetsResult>)adminClientResult).TrySetResult(
+                                                    new ListOffsetsResult() { ListOffsetsResultInfos = results.ListOffsetsResultInfos }
+                                                ));
+                                        break;
```

Comment:
Use `ListOffsetReport` here first and check if any error is present in it then throw ListOffsetsException